docs/PasswordInput.md

```diff
@@ -47,3 +57,22 @@ Set the [`autocomplete` attribute](https://developer.mozilla.org/en-US/docs/Web/
 <PasswordInput source="password" inputProps={{ autocomplete: 'current-password' }} />
 ```
 {% endraw %}
+
```

Comment:
We probably need to explain how to have 2 password inputs, and how to validate that they are equal.

Reply:
explained.